dd-sdk-android-internal/src/main/java/com/datadog/android/internal/DdCoreContentProvider.kt

```diff
@@ -64,16 +59,18 @@ class DdRumContentProvider : ContentProvider() {
         return 0
     }
 
-    internal companion object {
-        internal const val DEFAULT_IMPORTANCE: Int =
+    companion object {
+        const val DEFAULT_IMPORTANCE: Int =
             ActivityManager.RunningAppProcessInfo.IMPORTANCE_FOREGROUND
-        internal var processImportance = 0
+        var processImportance: Int = 0
+
+        val createTimeNs: Long = System.nanoTime()
```

Comment:
if I understand it right, the application start time will be a bit different now **only** for API 23? This may skew customer data a bit, but I guess the impact is very small both in terms of the affected devices and also in the value of data skew?